apps/speed.c

```diff
@@ -214,7 +214,7 @@ typedef enum OPTION_choice {
 } OPTION_CHOICE;
 
 const OPTIONS speed_options[] = {
-    {OPT_HELP_STR, 1, '-', "Usage: %s [options] [algorithm...]\n"},
+    {OPT_HELP_STR, 1, '-', "Usage: %s [options] [algorithm...]\nAll +int options consider prefix '0' as base-8 input, prefix '0x'/'0X' as base-16 input.\n"},
```

Comment:
This line has to be too long.  We wrap at 80 characters currently.  Breaking the string at the new line will help but isn't enough.  It needs a second break unfortunately.  I'd do it at the comma.